src/Core/Scheduler/Providers/WMS_Provider.js

```diff
@@ -57,6 +63,13 @@ WMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer)
     layer.style = layer.style || '';
     layer.transparent = layer.transparent || false;
 
+    if (layer.projection == 'EPSG:3857') {
+        layer.options.tileMatrixSet = 'PM';
```

Comment:
From an outsider look this looks wrong: why WMS would need knowledge of tileMatrixSet?

Reply:
Previously the `tileMatrixSet` was implicit because the projections were identical. Now, it's necessary to specify in which `tileMatrixSet` the textures are placed